Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM debian:jessie
+FROM gliderlabs/alpine:3.3
+ENTRYPOINT ["/bin/resolve-ip"]
 
-COPY bin/resolve-ip /usr/bin/resolve-ip
-COPY GeoLiteCity /usr/bin/GeoLiteCity
+COPY GeoLiteCity /bin/GeoLiteCity
 
-WORKDIR /usr/bin
+RUN apk-install ca-certificates
+COPY build/resolve-ip /bin/resolve-ip
+
+CMD ["--addr=0.0.0.0:80"]
```

Comment:
nit, but I would change the CMD here to be `["/bin/resolve-ip", "--addr=0.0.0.0:80"]` and remove the ENTRYPOINT line (see https://clever.atlassian.net/browse/CREW-47 and https://clever.atlassian.net/browse/CREW-48)